Makefile.core.mk

```diff
@@ -357,7 +357,7 @@ gen-check: gen check-clean-repo
 
 # Generate kustomize templates.
 gen-kustomize:
-	helm template -n istio-base manifests/base > manifests/base/files/gen-istio-cluster.yaml
+	helm template -n istio-base manifests/base -f manifests/global.yaml > manifests/base/files/gen-istio-cluster.yaml
```

Comment:
I would like to keep it as it was, without using the global.yaml - this way it can work naturally with helm(2|3), and avoids the current mess where globals is just a spaghetti dumping ground.

We may end up with few settings that are present in multiple templates - but that's fine as long as the names are consistent.

Reply:
sounds good, ill move just what we need (which may be nothing, if we get rid of the mtls stuff)